codalab/apps/chahub/tests/test_chahub_mixin.py

```diff
@@ -36,6 +36,9 @@ def setUp(self):
     def tearDown(self):
         settings.PYTEST_FORCE_CHAHUB = False
 
+    # It seems if we don't want these tests to pass without actual chahub settings, either these tests should get passed
+    # or if not set we override them to force tests. For now overriding settings so these tests run.
+    @override_settings(CHAHUB_API_URL='http://test_chahub.com', CHAHUB_API_KEY='123456789', CHAHUB_PRODUCER_ID=1)
     def test_submission_mixin_save_doesnt_resend_same_data(self):
```

Comment:
Why not change the test settings, why override many times?

Reply:
Moved to pytest.ini. If this isn't where we'd like to do it, let me know what you're thinking.